app/server/package.json

```diff
@@ -12,11 +12,11 @@
         "lint": "eslint \"src/**\"",
         "prisma-db-push": "prisma db push --skip-generate",
         "prisma-db-seed": "prisma db seed",
-        "test:prisma-db-push": "dotenv -c test yarn prisma-db-push",
+        "pretest": "yarn g:start-test-db && yarn prisma-db-push && yarn prisma-db-seed",
         "test:unit": "dotenv -c test -- jest \"src/.*\\.unit\\.test\\.ts\" --detectOpenHandles --runInBand",
         "test:integration": "dotenv -c test -- jest \"src/.*\\.integration\\.test\\.ts\" --detectOpenHandles --runInBand",
         "test:coverage": "dotenv -c test -- jest --coverage",
-        "test:ci": "yarn g:start-test-db && yarn test:prisma-db-push && yarn test:unit && yarn test:integration"
+        "test:ci": "yarn pretest && yarn test:unit && yarn test:integration"
```

Comment:
I think the documentation needs to be updated?

Reply:
I thought I did update it for this, but I can double check.